Source/abci/CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ file(GLOB sources *.cpp *.h Foundation/*.h Foundation/*.cpp Importer/*.h Importe
 list(APPEND sources ${ISPC_SOURCES})
 list(APPEND sources ${ISPC_OBJECTS})
 
-if (CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
+message(STATUS "The value of CMAKE_GENERATOR_PLATFORM is: ${CMAKE_GENERATOR_PLATFORM}")
```

Comment:
Do you want to keep this log? (I'm okey with leaving it here, just want to double check)